web/style/slider.scss

```diff
@@ -0,0 +1,97 @@
+.slideshow-container img {
+    margin-left: auto;
+    margin-right: auto;
+    display: block;
+  }
+  .mySlides {display: none}
+  img {vertical-align: middle;}
+  
+  /* Slideshow container */
+  .slideshow-container {
+   max-width: 1000px;
+   position: relative;
+   margin: auto;
+  }
+  
+  /* Next & previous buttons */
+  .prev, .next {
+   cursor: pointer;
+   position: absolute;
+   top: 50%;
+   width: auto;
+   padding: 16px;
+   margin-top: -22px;
+   color: #000;
+   font-weight: bold;
+   font-size: 1rem;
+   transition: 0.6s ease;
+   border-radius: 0 3px 3px 0;
+   user-select: none;
+  }
+  
+  /* Position the "next button" to the right */
+  .next {
+   right: 0;
+   border-radius: 3px 0 0 3px;
+  }
+  
+  /* On hover, add a black background color with a little bit see-through */
+  .prev:hover, .next:hover {
+   background-color: rgba(0,0,0,0.8);
+   color:$background-color-primary;
+  }
+  
+  /* Caption text */
+  .text {

```

Comment:
Caption text is carousel's content, it could very in size based on where its used, so it should not be part of this css module.